src/Marten.Testing/DevelopmentModeRegistry.cs

```diff
@@ -10,7 +10,10 @@ public class DevelopmentModeRegistry : Registry
         public DevelopmentModeRegistry()
         {
             For<IConnectionFactory>().Use<ConnectionSource>();
-            ForSingletonOf<IDocumentSchema>().Use<Marten.Schema.DocumentSchema>();
+            ForSingletonOf<DocumentSchema>()
+                .Use<DocumentSchema>()
+                .OnCreation("", (ctx, schema) => schema.UsePostgres9_4Fallback = true);
```

Comment:
currently configured to always run in fallback mode which at least demonstrates the fallback mechanism works...but keeps tests from demonstrating that the 9.5 functionality works.